doc/rados/operations/health-checks.rst

```diff
@@ -845,23 +846,24 @@ Data health (pools & placement groups)
 PG_AVAILABILITY
 _______________
 
-Data availability is reduced, meaning that the cluster is unable to
-service potential read or write requests for some data in the cluster.
-Specifically, one or more PGs is in a state that does not allow IO
-requests to be serviced.  Problematic PG states include *peering*,
-*stale*, *incomplete*, and the lack of *active* (if those conditions do not clear
-quickly).
+Data availability is reduced: in other words, the cluster is unable to service
+potential read or write requests for at least some data in the cluster.  More
+precisely, one or more Placement Groups (PGs) are in a state that does not
+allow I/O requests to be serviced. Any of the following conditions can count as
+problematic PG states if they do not clear quickly: *peering*, *stale*,
+*incomplete*, and the lack of *active*.
 
-Detailed information about which PGs are affected is available from:
+For detailed information about which PGs are affected, run the following
+command:
 
 .. prompt:: bash $
 
    ceph health detail
 
-In most cases the root cause is that one or more OSDs is currently
-down; see the discussion for ``OSD_DOWN`` above.
+In most cases, the root cause of this issue is that one or more OSDs are
+currently "down": see ``OSD_DOWN`` above.
```

Comment:
backticks instead of doublequotes, literal state.

Reply:
unfinished business--this use/mention confusion is general throughout the RADOS docs. A future project will rectify this. In this case, I suspect that you're right: the literal state will help the reader.